aksel.nav.no/website/pages/templates/skjemavalidering/demo.tsx

```diff
@@ -0,0 +1,175 @@
+    setTransportmiddelError(transportmiddelInvalid);
+    if (transportmiddelInvalid) {
+      isValidForm = false;
+    }
+
+    if (isValidForm) {
+      setFormSubmitted(true);
+    }
+  }
+
+  useEffect(() => {
+    errorSummaryRef.current?.focus();
+  }, [hasTriedToSubmit]);
+
+  const showErrorSummary = Boolean(
+    hasTriedToSubmit && (personnummerError || transportmiddelError),
+  );
+
+  if (formSubmitted)
+    return (
+      <Page style={{ marginTop: 20 }}>
+        <Page.Block as="main" width="lg" gutters>
+          <VStack gap="8" align="center">
+            <Heading size="large">Demo slutt</Heading>
+            <Button
+              onClick={() => {
+                location.hash = "";
+                location.reload();
+              }}
+            >
+              Nullstill
+            </Button>
+          </VStack>
+        </Page.Block>
+      </Page>
+    );
+
+  return (
+    <Page style={{ marginTop: 20 }}>
+      <Page.Block as="main" width="lg" gutters>
+        <form onSubmit={onSubmit}>
+          <VStack gap="8">
+            <TextField
+              id="personnummer"
+              label="Personnummer"
+              value={personnummer}
+              onChange={(e) => setPersonnummer(e.currentTarget.value)}
+              onBlur={() => {
+                Boolean(hasTriedToSubmit) &&
+                  setPersonnummerError(validatePersonnummer(personnummer));
+              }}
+              error={personnummerError}
+            />
+            <RadioGroup
+              id="transportmiddel"
+              tabIndex={-1}
+              legend="Transportmiddel"
+              value={transportmiddel}
+              onChange={(newValue) => {
+                setTransportmiddel(newValue);
+                setTransportmiddelError(false);
+              }}
+              error={transportmiddelError && "Du må velge et transportmiddel."}
+            >
+              <Radio value="car">Bil</Radio>
+              <Radio value="walking">Gange</Radio>
+              <Radio value="public">Kollektivtransport</Radio>
+            </RadioGroup>
+
+            {showErrorSummary && (
+              <ErrorSummary
+                ref={errorSummaryRef}
+                heading="Du må fikse disse feilene før du kan fortsette:"
+              >
+                {personnummerError ? (
+                  <ErrorSummary.Item href="#personnummer">
+                    {personnummerError}
+                  </ErrorSummary.Item>
+                ) : null}
+                {transportmiddelError && (
+                  <ErrorSummary.Item href="#transportmiddel">
+                    Du må velge et transportmiddel.
+                  </ErrorSummary.Item>
+                )}
+              </ErrorSummary>
+            )}
+
+            <HGrid
+              gap={{ xs: "4", sm: "8 4" }}
+              columns={{ xs: 1, sm: 2 }}
+              width={{ sm: "fit-content" }}
+              style={{ marginBottom: 200 }}
```

Comment:
```suggestion
              marginBlock="0 32"
              paddingBlock="0 32"
```

This technically achieves the same thing, but not sure if its an improvement

Reply:
I don't think we need this, actually 🤔